core/jazz_ui/src/config/templates/go-yaml.ts

```diff
@@ -2,7 +2,7 @@ export const goTemplate = {
     template:
 `functions: 
     function1: 
-      handler: function1/main
+      functions/function1/main
```

Comment:
It should be      handler: functions/function1/main

Reply:
made this change